typings/externs.d.ts

```diff
@@ -59,13 +59,19 @@ declare global {
 
     interface ThrottlingSettings {
       // simulation settings
+      // The round trip time in milliseconds
```

Comment:
I believe @brendankenny showed me that doing `/** <comment>` lets IDEs interpret it as a JSDoc comment on usages :)